charts/iap/templates/configmaps.yaml

```diff
@@ -20,6 +20,9 @@ metadata:
   name: iap-{{ .name }}-configmap
 data:
   config.toml: |
+{{- if eq $.Values.iap.approval_prompt "none"}}
```

Comment:
What happens if I set `approval_prompt: force` in my values.yaml? Will we then just ignore the field?